modules/oc-mirror-about-cache-and-workspace-dirs.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oc-mirror-about-cache-and-workspace-dirs_{context}"]
+= About the --cache-dir and --workspace flags
+
+You can use the `--cache-dir` flag to specify a directory where the oc-mirror plugin stores a persistent cache of image blobs and manifests for use during mirroring operations.
+
+The oc-mirror plugin uses the cache in the `disk-to-mirror` and `mirror-to-disk` workflows but does not use the cache in the `mirror-to-mirror` workflow. The plugin uses the cache to perform incremental mirroring and avoids remirroring unchanged images, which saves time and reduces network bandwidth usage.
+
+The cache directory only contains data up to the last successful mirroring operation. If you delete or corrupt the cache directory, the oc-mirror plugin pulls the image blobs and manifests again, which can force a full remirror and increase network usage.
+
+// You can use the --workspace flag to specify a directory where the oc-mirror plugin stores the working files it creates during a mirroring operation, such as the ImageDigestMirrorSet and ImageTagMirrorSet manifests.
```

Comment:
Delete this comment if it's not going to be included in the file. Is a future addition for this line planned?